dspace-server-webapp/src/test/java/org/dspace/app/rest/ItemRestRepositoryIT.java

```diff
@@ -197,6 +197,15 @@ public void findAllWithPaginationTest() throws Exception {
                                       .withSubject("ExtraEntry")
                                       .build();
 
+        // Create a Workspace Item (which in turn creates an Item with "in_archive=false")
+        // This is only created to prove that WorkspaceItems are NOT counted/listed in this endpoint
+        WorkspaceItem workspaceItem = WorkspaceItemBuilder.createWorkspaceItem(context, col2)
+                .withTitle("In Progress Item")
+                .withIssueDate("2018-02-05")
+                .withAuthor("Doe, Jane").withAuthor("Smith, Jennifer")
+                .build();
+        Item itemInWorkspace = workspaceItem.getItem();
+
```

Comment:
it would be more effective to create also a workflow item and a template item just to proof that nothing else is counted

Reply:
done!  Added tests for those in the latest commit & definitely a good suggestion